### PR TITLE
Playground: ExtrusionsTest

### PR DESCRIPTION
**Pull request by Gustl22**
Named Extrusions and not S3DB as it also can be used for regular extrusions, too, if needed.

### CODE REVIEW

File: vtm-playground/src/org/oscim/test/ExtrusionsTest.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+import org.oscim.layers.tile.buildings.BuildingLayer;
+import org.oscim.layers.tile.buildings.S3DBLayer;
+import org.oscim.layers.tile.vector.VectorTileLayer;
+import org.oscim.test.tiling.source.TestTileSource;
+import org.oscim.theme.VtmThemes;
+
+public class ExtrusionsTest extends GdxMapApp {
+
+    enum GroundShape {
+        Rect, Hexagon, L, O,
```
Comment (devemux86) [inline vtm-playground/src/org/oscim/test/ExtrusionsTest.java]: `L`, `O` ?

Comment (Gustl22) [reply vtm-playground/src/org/oscim/test/ExtrusionsTest.java]: Describes the shape of letter compared to the ground. If you know a better term, you're welcome.

Comment (devemux86) [reply vtm-playground/src/org/oscim/test/ExtrusionsTest.java]: Didn't even understand what it is, thought Line and Outline.

Comment (Gustl22) [reply vtm-playground/src/org/oscim/test/ExtrusionsTest.java]: It describes the shape of the Letter: **L** and **O**. Search for [multi pitched](https://en.m.wikipedia.org/wiki/List_of_roof_shapes#Roof_shapes) roofs.

Comment (devemux86) [reply vtm-playground/src/org/oscim/test/ExtrusionsTest.java]: For clarity better use full words:
```java
HEXAGON, RECTANGLE, SHAPE_L, SHAPE_O
```

Comment (Gustl22) [reply vtm-playground/src/org/oscim/test/ExtrusionsTest.java]: Done.

**Comment by devemux86** [comment]
Thanks, merged via 73e17ef4a1cea11625063e81fd50f35a4f02c901.

**Pull request closed without merging**